helios-services/src/main/java/com/spotify/helios/rollingupdate/RollingUpdateOpFactory.java

```diff
@@ -195,6 +195,8 @@ public RollingUpdateOp error(final String msg, final String host,
         deploymentGroup, task, errMsg, errorCode, metadata);
     events.add(taskEv);
     events.add(eventFactory.rollingUpdateFailed(deploymentGroup, taskEv));
+    log.info("rolling-update step on deployment-group name={} failed; message={}",
```

Comment:
might be nice if this could mention the job too - something to identify _which_ rolling-update on the group it was - but seems like that might not be available in this method

Reply:
At least for the cases I've looked at, `errMsg` includes that information.

https://github.com/spotify/helios/blob/master/helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java#L846